src/QuickMarcEditor/constants.js

```diff
@@ -99,3 +99,5 @@ export const HOLDINGS_FIXED_FIELD_DEFAULT_VALUES = {
 };
 
 export const CORRESPONDING_HEADING_TYPE_TAGS = ['100', '110', '111', '151', '130', '150', '155'];
+
+export const TAGS_FOR_DISPLAYING_LINKS = new Set(['100', '110', '111', '700', '710', '711']);
```

Comment:
Is that necessary to use `Set` over an array here?

Reply:
I ramp down the complexity of the algorithm, however, for this amount of data, this is not a big boost.